core/app/c/[communitySlug]/pubs/[pubId]/page.tsx

```diff
@@ -114,15 +106,16 @@ export default async function Page({
 				<div className="flex-1">
 					{pub.values
 						.filter((value) => {
-							return value.field.name !== "Title";
+							return value.fieldName !== "Title";
```

Comment:
I think we may have an easier way to figure out the title. this is more of a suggestion for the full pr, but if here

https://github.com/pubpub/platform/blob/9ac60fd743e4403008fc7ea5fe29f478a0db6387/core/lib/server/pubtype.ts#L34

you add `_PubFieldToPubType.isTitle`, you should get title information about the pubtype of pubs, which you should be able to use here to filter out the actual title.

Reply:
ah okay that works! although that puts it on `pubType` when ideally it'd be on the `values` so we can still pass it to `getPubTitle`. I'm thinking `getPubTitle` probably can just take a `pubType` with fields (would have to merge for the value though...) but I didn't want to do that here since the func is used in a few places

Reply:
yeah that's fine, i will tackle that in #736